tests/test_install.py

```diff
@@ -14,21 +14,20 @@
 from conda_pypi.python_paths import get_env_python, get_env_site_packages
 
 
-@pytest.mark.skip(reason="Migrating to alternative install method using conda pupa")
 @pytest.mark.parametrize(
     "pypi_spec,conda_spec,channel",
     [
         ("numpy", "", "conda-forge"),
         ("numpy=1.20", "", "conda-forge"),
-        # build was originally published as build in conda-forge
-        # and later renamed to python-build; conda-forge::build is
-        # only available til 0.7, but conda-forge::python-build has 1.x
-        ("build>=1", "python-build>=1", "conda-forge"),
-        # ib-insync is only available with dashes, not with underscores
-        ("ib_insync", "ib-insync", "conda-forge"),
-        # these won't be ever published in conda-forge, I guess
-        ("aaargh", None, "pypi"),
-        ("5-exercise-upload-to-pypi", None, "pypi"),
+        # # build was originally published as build in conda-forge
+        # # and later renamed to python-build; conda-forge::build is
+        # # only available til 0.7, but conda-forge::python-build has 1.x
+        # ("build>=1", "python-build>=1", "conda-forge"),
+        # # ib-insync is only available with dashes, not with underscores
+        # ("ib_insync", "ib-insync", "conda-forge"),
+        # # these won't be ever published in conda-forge, I guess
+        # ("aaargh", None, "pypi"),
+        # ("5-exercise-upload-to-pypi", None, "pypi"),
```

Comment:
Can we please just remove these test cases if we don't want to run them any more?